README.md

```diff
@@ -4,8 +4,28 @@
 This is a void implementation of PSR-6. Other names for this adapter could be Blackhole or Null. This 
```

Comment:
I tested Shields.io just now. It is way slow and seams to fail. 
![screen shot 2016-01-25 at 16 53 14](https://cloud.githubusercontent.com/assets/1275206/12556016/2ea0a5d4-c384-11e5-8e16-a4983ed7952b.png)

I would suggest to copy from the redis adapter and make a search replace.
```html
[![Latest Stable Version](https://poser.pugx.org/cache/redis-adapter/v/stable)](https://packagist.org/packages/cache/redis-adapter) [![codecov.io](https://codecov.io/github/php-cache/redis-adapter/coverage.svg?branch=master)](https://codecov.io/github/php-cache/redis-adapter?branch=master) [![Build Status](https://travis-ci.org/php-cache/redis-adapter.svg?branch=master)](https://travis-ci.org/php-cache/redis-adapter) [![Total Downloads](https://poser.pugx.org/cache/redis-adapter/downloads)](https://packagist.org/packages/cache/redis-adapter)  [![Monthly Downloads](https://poser.pugx.org/cache/redis-adapter/d/monthly.png)](https://packagist.org/packages/cache/redis-adapter) [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
```